app/assets/v2/js/pages/dashboard.js

```diff
@@ -290,9 +290,14 @@ var get_search_URI = function(offset) {
 
   var order_by = localStorage['order_by'];
 
-  if (order_by) {
-    uri += '&order_by=' + order_by;
+  if (!document.hackathon) {
+    if (order_by != 'undefined') {
```

Comment:
I think only `if (order_by)` works better in this case since it will check for all possible falsy values.

Reply:
@SaptakS I had the same thought until i hit `'undefined'` 😄